src/common/client-app.ts

```diff
@@ -69,8 +74,8 @@ export class App {
 
   onInterval= () => {
     var current = this.boardVM.toPlain();
-    var myChanges = rfc6902.createPatch(this.shadow, current);
-    if (myChanges.length) {
+    var myChanges = rfc6902.createPatch(this.shadow, current);
+    if (myChanges.length) {
```

Comment:
Is it not missed to apply `myChanges` to a shadow? if not `shadow` and `serverShadow` will be always the same? does it make sense?